src/core/utilities/identifiers.test.ts

```diff
@@ -0,0 +1,43 @@
+import { validateDSNPId } from "./identifiers";
+
+describe("validateDSNPId", () => {
```

Comment:
This would be more readable as a table test.

Reply:
Good idea. Fixed